vmm/sandbox/src/main.rs

```diff
@@ -108,7 +106,7 @@ async fn main() -> anyhow::Result<()> {
         .format_timestamp_micros()
         .init();
     #[cfg(feature = "qemu")]
-    let sandboxer: KuasarSandboxer<QemuVMFactory, QemuHooks> = {
+    let _sandboxer: KuasarSandboxer<QemuVMFactory, QemuHooks> = {
```

Comment:
These sandboxers are used in different features, use`#[allow(dead_code)]`

Reply:
Already added #[allow(dead_code)]